src/features/adr/pages/ADRForm.tsx

```diff
@@ -1,78 +1,61 @@
-import { Button, Paper, Skeleton } from "@material-ui/core";
-import { JsonSchemaForm } from "../../../shared/components";
-import { FC, useRef, useState } from "react";
+import { Skeleton } from "@material-ui/core";
+import { FC } from "react";
 import { useADRSchemaQuery } from "./hooks/useADRSchemaQuery";
-import { stylesFactory } from "../../../shared/utils";
 import {
-  useIsSchemaValid,
   useLoadingSpinner,
   useTranslations,
+  useModal,
 } from "../../../shared/hooks";
 import { useSubmitADR } from "./hooks";
 import { Box } from "@material-ui/core";
-
-const useStyles = stylesFactory({
-  img: {
-    display: "flex",
-    marginLeft: "auto",
-    marginRight: "auto",
-    maxWidth: "100%",
-  },
-
-  paper: {
-    padding: 20,
-    maxWidth: 600,
-    minHeight: "100vh",
-    display: "flex",
-    alignItems: "flex-end",
-    marginLeft: "auto",
-    marginRight: "auto",
-  },
-});
+import { Stepper } from "../../../shared/components/stepper/Stepper";
+import { StepperForm } from "../../../shared/components/stepper/StepperForm";
+import { PatientForm } from "./PatientForm";
+import { ModalKey } from "../../../shared/containers/ModalProvider";
 
 export const ADRForm: FC = () => {
   const { messages } = useTranslations();
-  const classes = useStyles();
-  const submitRef = useRef<HTMLButtonElement | null>(null);
   const { isLoading, data } = useADRSchemaQuery();
-  const [formData, setFormData] = useState({});
   const { toggleLoading } = useLoadingSpinner();
-  const isValid = useIsSchemaValid(data?.jsonSchema ?? {}, formData);
   const { submit } = useSubmitADR();
+  const { open, close } = useModal(ModalKey.confirm);
+  const handleClose = () => {
+    close();
+    window.location.reload();
```

Comment:
using this to reset the form state on submit - usual use case is to only submit one ADR, so no need to be quicker.
Doing it this way means that you can go back to step one and pick another person without searching again - the other option would be to control the state of a child component on submit which I thought was messier.